hocs/withAuthorization.tsx

```diff
@@ -0,0 +1,54 @@
+import Unauthorized from 'components/PageComponents/Unauthorized';
+import { useAuth } from 'hooks/useAuth';
+import { UserGroup } from 'lib/types';
+import { ContextUserType } from 'lib/types/auth';
+import { useRouter } from 'next/router';
+
+const editorIds = ['Kx00tfTGy6ei8olseVTJc988f992'];
```

Comment:
Where is the id here coming from? Just an "example editor"?

Reply:
yes

Reply:
while we're still testing, we can put our test user ids here and have access to all pages. Will have to be removed when we go live.